src/volume.ts

```diff
@@ -98,6 +99,30 @@ export class Volume {
     });
   }
 
+  public static fromSecret(secret: ISecret, options: SecretVolumeOptions = { }): Volume {
```

Comment:
Please add unit tests for this function in `volume.test.ts`

Reply:
Added tests in test/volume.test.ts

Reply:
@ambasta Sorry I didn't catch this earlier - but can you also add a docstring to this `fromSecret` function? See `fromConfigMap` as a reference

Reply:
I am surprised I missed this as well. I was sure I'd added this before, but pushed an updated commit to address this. Let me know if I should squash the two commits into one.